<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Find</class>
 <widget class="QDialog" name="Dialog_Find">
  <property name="modal">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>194</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <item row="4" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Direction</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <widget class="QRadioButton" name="down_RB">
        <property name="text">
         <string>Down</string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="up_RB">
        <property name="text">
         <string>Up</string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="searchTop_CKB">
     <property name="text">
      <string>Search from top of Log File</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="verticalSpacer_1">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>10</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="10" column="2">
    <spacer name="verticalSpacer_2">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="case_CKB">
     <property name="text">
      <string>Match Case</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Find:</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="find">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <spacer name="horizontalSpacer_21">
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="find_PB">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_23">
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>25</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_PB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_22">
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>case_CKB</tabstop>
  <tabstop>searchTop_CKB</tabstop>
  <tabstop>up_RB</tabstop>
  <tabstop>down_RB</tabstop>
  <tabstop>find_PB</tabstop>
  <tabstop>cancel_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
